server/application/application.go

```diff
@@ -1988,13 +1990,19 @@ func (s *Server) getUnstructuredLiveResourceOrApp(ctx context.Context, rbacReque
 		if err != nil {
 			return nil, nil, nil, nil, fmt.Errorf("error getting application cluster config: %w", err)
 		}
+		data, _ := json.Marshal(obj)
```

Comment:
We should probably handle the error here.

Reply:
Was temporarily used use for debugging printouts, now gone.